docs/yaml_configuration.rst

```diff
@@ -300,7 +300,7 @@ PostgreSQL
    -  **pgpass**: path to the `.pgpass <https://www.postgresql.org/docs/current/static/libpq-pgpass.html>`__ password file. Patroni creates this file before executing pg\_basebackup, the post_init script and under some other circumstances. The location must be writable by Patroni.
    -  **recovery\_conf**: additional configuration settings written to recovery.conf when configuring follower.
    -  **custom\_conf** : path to an optional custom ``postgresql.conf`` file, that will be used in place of ``postgresql.base.conf``. The file must exist on all cluster nodes, be readable by PostgreSQL and will be included from its location on the real ``postgresql.conf``. Note that Patroni will not monitor this file for changes, nor backup it. However, its settings can still be overridden by Patroni's own configuration facilities - see :ref:`dynamic configuration <patroni_configuration>` for details.
-   -  **parameters**: list of configuration settings for Postgres. Many of these are required for replication to work.
+   -  **parameters**: set of configuration settings for Postgres in format ``{ssl: on, ssl\_cert\_file: cert\_file}``. Many of these are required for replication to work.
```

Comment:
Well, while technically this is almost a valid YAML, IMO it doesn't add a lot of clarity.
Starting from the fact that curly braces in YAML are used very rarely, and by adding on top of that that `on` is evaluated as boolean, while we need a string.
Moreover, backslash escapes here are redundant, and make rendered page look incorrectly.

Reply:
Thanks for your review.

I follow the `labels` option under https://patroni.readthedocs.io/en/latest/yaml_configuration.html#kubernetes to use brace to indicate that this is a dict in python. I think given a yaml snippet as example is better actually, if it's unnecessary to follow the style of `labels`.

Reply:
```suggestion
   -  **parameters**: configuration parameters (GUCs) for Postgres in format ``{ssl: "on", ssl_cert_file: "cert_file"}``. Many of these are required for replication to work.
```

As I said, backslash escapes are incorrectly rendered.